pyscriptjs/src/plugins/fetch.ts

```diff
@@ -14,7 +14,10 @@ export function calculatePaths(fetch_cfg: FetchConfig[]) {
         {
             if (to_file !== undefined)
             {
-                throw new UserError(`Cannot use 'to_file' and 'files' parameters together!`);
+                throw new UserError(
+                    UserError.ErrorCode.FETCH_PARAMETER_ERROR,
+                    `Cannot use 'to_file' and 'files' parameters together!`
+                );
```

Comment:
this seems more a `BAD_CONFIG` error, not a fetch error (we didn't even try to fetch anything).
Also, when the fetch logic becomes a "proper" plugin, this should probably be checked inside the `configure` hook (and then it becomes even more evident that it's a `BAD_CONFIG` error)